jzaki/bls-wallet
# updated default env to use arbitrum nitro goerli

## What is this PR doing?
Defaults the network of extension and aggregator to arbitrum nitro goerli chain

## How can these changes be manually tested?
- clear existing postgres container
- build extension 
- import extension, create new wallet and send tx
- the tx should be sent on nitro

## Does this PR resolve or contribute to any issues?
#310 

## Checklist

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)
